embassy-stm32/src/rcc/bd.rs

```diff
@@ -235,6 +250,15 @@ impl LsConfig {
             });
 
             while !bdcr().read().lserdy() {}
+
+            #[cfg(any(rcc_l5, rcc_u5, rcc_wle, rcc_wl5, rcc_wba))]
+            if let Some(lse_sysen) = lse_sysen {
+                bdcr().modify(|w| {
+                    w.set_lsesysen(lse_sysen);
+                });
+
+                while !bdcr().read().lsesysrdy() {}
```

Comment:
maybe it's this loop that hangs if lse_sysen=false.

Reply:
Ah yes. I can only wait for lsesysrdy when I set lsesyson=1. When the user chooses lsesys=0 I can't wait for this. Will fix this later